Cargo.toml

```diff
@@ -62,7 +62,7 @@ wots_deprecated_do_not_use = [
 sha = [ "sha2", "digest" ]
 hmac = [ "dep:hmac" ]
 pbkdf = [ "pbkdf2" ]
```

Comment:
Shall we require also "hmac" and "sha" deps in "pbkdf" feature? Without those features pbkdf2_hmac_sha won't work.

Reply:
Yeah